tests/modellauncher.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0"?>
+<pecan>
+  <outdir>pecan</outdir>
+
+  <database>
+    <bety>
+      <user>bety</user>
+      <password>bety</password>
+      <host>postgres</host>
+      <dbname>bety</dbname>
+      <driver>PostgreSQL</driver>
+      <write>FALSE</write>
+    </bety>
+    <dbfiles>/data/dbfiles/dbfiles</dbfiles>
+  </database>
+
+  <host>
+    <name>localhost</name>
+    <modellauncher>
+      <binary>/usr/local/bin/modellauncher</binary>
+      <qsub.extra>-pe omp 20</qsub.extra>
+    </modellauncher>
+  </host>
+
+  <pfts>
+    <pft>
+      <name>temperate.coniferous</name> 
+    </pft>
+  </pfts>
+
+  <meta.analysis>
+    <iter>3000</iter>
+    <random.effects>FALSE</random.effects>
+    <threshold>1.2</threshold>
+    <update>AUTO</update>
+  </meta.analysis>
+
+  <ensemble>
+    <size>200</size>
+    <variable>NPP</variable>
+  </ensemble>
+
+  <model>
+    <binary>/usr/local/bin/sipnet.runk</binary>
```

Comment:
same here

Reply:
This is the location of the model that needs to be exectued on the cluster.

Reply:
I get that, my question was whether this test will run if the modellauncher or binary located somewhere else on the specific machine, like modex.  But this may just be a test so perhaps doesnt matter since during the run the user would set these paths to the correct location.

Reply:
correct this is just a test for me, and an example for others to see how to configure it. You will need to modify this for your specific environment.